deploy/helm-charts/server/templates/policy_fail_vw.yaml

```diff
@@ -0,0 +1,63 @@
+{{- if eq .Values.webhook.failurePolicy "Fail" }}
```

Comment:
Had to create this file just to support `validatingwebhookconfiguration` failurePolicy to be `FAIL`.

It turns out, webhook doesn't allow the terrascan server pod to come up in case `failurePolicy` is `Fail`.

So, as a workaround, we create the webhook w/ `Ignore`, and then upgrade it to `Fail` in. post install chart hook. ref: https://v2.helm.sh/docs/charts_hooks/

Reply:
> Had to create this file just to support validatingwebhookconfiguration failurePolicy to be FAIL.

It turns out, webhook doesn't allow the terrascan server pod to come up in case failurePolicy is Fail.

So, as a workaround, we create the webhook w/ Ignore, and then upgrade it to Fail in. post install chart hook. ref: https://v2.helm.sh/docs/charts_hooks/

Can you add this as a comment in the vw yaml file as well?

Reply:
sure